services/youtube/youtube-likes.service.js

```diff
@@ -62,11 +62,14 @@ export default class YouTubeLikes extends YouTubeVideoBase {
       id,
     })
     if (queryParams && typeof queryParams.withDislikes !== 'undefined') {
+      const likes = `${metric(statistics.likeCount)} 👍`
+      const dislikes =
+        statistics.dislikeCount !== undefined
+          ? `${metric(statistics.dislikeCount)} 👎`
+          : ''
       renderedBadge = {
         ...renderedBadge,
-        message: `${metric(statistics.likeCount)} 👍 ${metric(
-          statistics.dislikeCount
-        )} 👎`,
+        message: `${likes} ${dislikes}`,
```

Comment:
We already have the `withDislikes` query parameter that allows the user to control whether or not the dislikes are included on their badge. If we hit this block then the user has explicitly asked for the dislike portion to be included as well, and as such we should instead display `0 👎`.

Perhaps `${metric(statistics.dislikeCount || 0)} 👎` ?

Reply:
But `0 👎` looks like there is no 👎, while in fact this is unknown. I was choosing between not presenting this piece or using `? 👎`. I'm against using `0` for the case.

Reply:
> while in fact this is unknown.

What is your evidence for this? Do you have some documentation to the API or an example target that includes a `"dislikeCount": 0` in the response?

Reply:
Actually, https://techcrunch.com/2021/11/10/youtube-is-removing-the-dislike-count-on-all-videos-across-its-platform/

Youtube _has_ dropped the dislike count from public view altogether, and that's not something we'll ever be able to see. I think we should actually remove dislikes across the board

Reply:
I'd rather like this PR to only restore YT stats.
I think total dislikes removal could be the subject for the next one.

Reply:
We have a feature that was based on an attribute in the API response that no longer exists, which in turn has broken badges. I don't understand what you're suggesting be split, could you elaborate?

We need a resolution to the actual issue, i.e. our badge needs to account for the upstream change. Splitting that into multiple PRs with ~5 line diffs is unnecessary overhead for us

Reply:
> Splitting that into multiple PRs with ~5 line diffs is unnecessary overhead for us

Understood.

Removed code IMO related to the dislikes.
Locally tested that the old links, with (now unused) parameter `withDislikes` render the no-parameterized version fine.

Reply:
Awesome thanks so much! Will give this a final pass here shortly